### PR TITLE
[BUG] Modify Eslint rules for react/propTypes

### PR DESCRIPTION
**Pull request by David-Xia0**
added new ex lint rule and fixed eslint errors

### CODE REVIEW

**Review by BlinkCreator — APPROVED** [review]
Through the call we had you accurately justified the change to the es lint. Everything looks good too.

**Review by Tianrens — APPROVED** [review]

**Review by OverCry — APPROVED** [review]

File: frontend/src/components/ExampleTestButton/Button.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+ * @param {*} text
+ * @return {*}
+ */
 export default function Button({text}) {
-    return (
-        <div>
-            <button>{text}</button>
-        </div>
-    );
-}
```
Comment (Alxjlv) [inline frontend/src/components/ExampleTestButton/Button.js:9]: I'm guessing these changes are a result of running the linter?

**Review by Alxjlv — APPROVED** [review]
Looks good to me!

File: frontend/src/components/ExampleTestButton/Button.js (hunk 1 above)
Comment (David-Xia0) [reply frontend/src/components/ExampleTestButton/Button.js:9]: Yes

**Pull request merged**